controllers/productController.js

```diff
@@ -8,11 +8,24 @@ const createReview = async (req, res) => {
 
     if (!content || !productId) throw new appError('KEY_ERROR', 400);
 
-    await prodcutService.createReview(content, userId, productId);
+    await productService.createReview(content, userId, productId);
 
     res.status(201).json({ message: 'REVIEW_CREATED' });
 }
 
+const loadReviews = async (req, res) => {
```

Comment:
대부분 데이터를 호출하는 함수를 만들 때, `get`을 많이 사용하기 때문에 가독성 측면에서 더 좋을 것 같습니다.

```suggestion
const getReviews = async (req, res) => {
```